src/headers.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  SALEOR_AUTHORIZATION_BEARER_HEADER,
+  SALEOR_DOMAIN_HEADER,
+  SALEOR_EVENT_HEADER,
+  SALEOR_SIGNATURE_HEADER,
+} from "./const";
+
+export const getSaleorHeaders = (headers: { [name: string]: any }) => ({
```

Comment:
Why not

```suggestion
export const getSaleorHeaders = (headers: { [name: string]: string }) => ({
```

Reply:
I've followed the types from the Retes lib. I'll catch up with the authors what was the rationale behind any type and limit types in the separate PR